src/pages/Search/AdvancedSearchFilters.tsx

```diff
@@ -552,9 +552,14 @@ function AdvancedSearchFilters() {
         .map((section) => {
             return section
                 .map((key) => {
-                    // 'feed' filter row does not appear in advanced filters, it is created using selected cards
-                    // 'payer' and 'reimburser' do not appear in advanced filters, they are created using suggested searches
-                    if (key === CONST.SEARCH.SYNTAX_FILTER_KEYS.FEED || key === CONST.SEARCH.SYNTAX_FILTER_KEYS.EXPORTER || key === CONST.SEARCH.SYNTAX_FILTER_KEYS.PAYER) {
+                    // 'feed' filter row does not appear in advanced filters, it is created using selected cardsAdd commentMore actions
```

Comment:
Not sure what this means
```suggestion
                    // 'feed' filter row does not appear in advanced filters, it is created using selected cards
```